src/BitcoinHelpers.js

```diff
@@ -542,36 +542,14 @@ const BitcoinHelpers = {
      *
      * @param {string} bitcoinAddress Bitcoin address to check.
      *
-     * @return {Promise<Number>} A promise to the confirmed balance as satoshis.
+     * @return {Promise<Number>} A promise to the confirmed balance in satoshis.
      */
     getBalance: async function(bitcoinAddress) {
       return await BitcoinHelpers.withElectrumClient(async electrumClient => {
         const script = BitcoinHelpers.Address.toScript(bitcoinAddress)
         return (await electrumClient.getBalanceOfScript(script)).confirmed
       })
     },
-    /**
-     * Estimates the fee to have a transaction included within the specified
-     * number of blocks.
-     *
-     * @param {number} [includeWithinBlocks=1] - The estimate is based on having
-     *        the transaction included within this number of blocks. Defaults to
-     *        trying to include the transaction within 1 block.
-     *
-     * @return {Promise<number>} The estimated fee to execute the provided
-     *         transaction. Fails if the fee cannot be estimated.
-     */
-    estimateFeePerKb: async function(includeWithinBlocks) {
-      return BitcoinHelpers.withElectrumClient(async electrumClient => {
-        const feePerKb = await electrumClient.getFeeEstimate(includeWithinBlocks || 1)
-
-        if (feePerKb < 0) {
-          throw new Error(`Fee cannot be estimated; Electrum returned ${feePerKb}.`)
-        }
-
-        return feePerKb
-      })
-    },
```

Comment:
I'm definitely interested in making this work in the future, btw. Need to see what about our node is making the server side of this fall over.